test/Twig/Tests/LegacyIntegrationTest.php

```diff
@@ -22,6 +22,15 @@ public function getFixturesDir()
     {
         return dirname(__FILE__).'/LegacyFixtures/';
     }
+
+    public function getTests($name, $legacyTests = false)
+    {
+        if (!$legacyTests) {
+            return array(array('not', '-', '', array(), '', array()));
+        }
+
+        return parent::getTests($name, true);
+    }
```

Comment:
The test will no fail again if there are no fixtures to test (a minor BC which is reverted by this). 
The exception on that rule is now placed in the test that needs that exception and no longer in the parent.